src/components/VerboseTextField.tsx

```diff
@@ -41,20 +44,28 @@ export interface FormTextFieldProps extends OutlinedTextFieldProps {
  * @param props.state The status color of the text field.
  * @returns
  */
-export default function VerboseTextField(
-  props: Omit<FormTextFieldProps, "variant">
-) {
+export default function VerboseTextField(props: VerboseTextFieldProps) {
   const {
     state = undefined,
     description = undefined,
+    fieldName = undefined,
     formDescriptionTextProps = {},
   } = props;
 
   const labelColor = !state?.statusValue ? "primary" : state.statusValue;
 
   return (
     <>
+      {fieldName ? (
```

Comment:
fieldName && <..> is more common

Reply:
fixed in https://github.com/inrupt/client-identifier-helper/pull/102/commits/b91eaa149bb187e84117122a8ad802166bba7efc